app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -120,6 +122,13 @@ def index
 
     def index_relation
       query = Repository.readable
+      unless params[:docker_tag_id].nil?
```

Comment:
Something feels a bit off to me about the changes to this controller and the angular controller. I think it's because they are both treating DocketMetaTags as a special case. I want to throw out an idea for you, and you tell me if will help make things cleaner, worse, or whatever: I see we have a docker_tags_controller. Can we add a 'def repositories' method (ie /api/v2/docker_tags/:id/repositories) that knows what do do for docker tags specifically? Then we can use that from the UI controller

Reply:
Sure, I can't think of any immediate reason why that wouldn't work here. It is a cleaner and more reusable solution.